### DISCUSSION TITLE
This is such a cool library

### DISCUSSION THREAD

**Discussion by shrshk** [Pustur/whatsapp-chat-parser#240] [General]
I used this to remove inactive users from our pick up soccer group :)

**Comment by Pustur** [comment]
Hey Shirish,

I'm glad to hear that the library was useful to you! That's a use case I didn't think about when developing it.

Thanks for sharing your experience 😊

**Reply by shrshk** [reply]
https://github.com/adiwajshing/Baileys
https://github.com/pedroslopez/whatsapp-web.js

I also found these libraries very useful, all of these make WhatsApp automation a breeze, now that [GTP-3](https://openai.com/api/) is openly available, we can reply to any message automatically.
